### ISSUE TITLE
Magento 2.4 support?

### ISSUE THREAD

**Issue by boehsermoe** [netresearch/module-compatibility-m2#1]
Is the module still compatible with magento 2.4?

**Comment by powli** [comment]
No, 2.4 has not yet been considered for this module, as we ourselves do not actively use the module at the moment.
